addons/account_peppol/models/res_company.py

```diff
@@ -152,6 +153,17 @@ def _compute_peppol_purchase_journal_id(self):
             else:
                 company.peppol_purchase_journal_id = False
 
+    def _inverse_peppol_purchase_journal_id(self):
+        for company in self:
+            # This avoid having 2 or more journals from the same company with 
+            # `is_peppol_journal` set to True (which could occur after changes).
+            journals_to_reset = self.env['account.journal'].search([
+                ('company_id', '=', company.id),
+                ('is_peppol_journal', '=', True),
+            ])
+            journals_to_reset.write({'is_peppol_journal': False})
```

Comment:
```suggestion
           journals_to_reset.is_peppol_journal = False
```
this works with multiple records too

Reply:
Nice didn't know! Thanks :smile: